modules/setup/application/forms/LdapDiscoveryPage.php

```diff
@@ -74,6 +74,8 @@ public function isValid($data)
                 $this->discovery = Discovery::discoverDomain($data['domain']);
                 if ($this->discovery->isSuccess()) {
                     return true;
+                } else {
+                    $this->getElement('domain')->addError($this->discovery->getError()->getMessage());
```

Comment:
The error message should not be bound to the domain field. It should be a form error instead.